packages/conventional-changelog-core/test/test.js

```diff
@@ -1,48 +1,47 @@
 'use strict'
 const conventionalChangelogCore = require('../')
 const expect = require('chai').expect
-const mocha = require('mocha')
-const describe = mocha.describe
-const it = mocha.it
 const gitTails = require('git-tails').sync
-const shell = require('shelljs')
-const gitDummyCommit = require('git-dummy-commit')
```

Comment:
`git-dummy-commit` indirrectly uses shelljs and does not seem to be maintained 

there is PR that move this package to better state but it doesn't seem that will get merged soon
https://github.com/stevemao/git-dummy-commit/pull/3

https://github.com/stevemao/git-dummy-commit